code/src/components/Input3.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+const optionsQuestionThree = ['< 20', '20-25', '25 <']
```

Comment:
I think it could have been nice to write 25 + instead of 25 < since this is not something most non-coders might use in everyday life when talking about temperatures.